full-stack-tests/presentation/src/IModelSetupUtils.ts

```diff
@@ -19,6 +19,7 @@ export function createValidIModelFileName(imodelName: string) {
 export async function buildTestIModelDb(name: string, cb: (db: IModelDb) => Promise<void>) {
   const outputFile = setupOutputFileLocation(createValidIModelFileName(name));
   const db = SnapshotDb.createEmpty(outputFile, { rootSubject: { name } });
+  db.channels.addAllowedChannel(ChannelControl.sharedChannelName);
```

Comment:
Looking at changes in `IModelDb.ts` I see that this is being called in `SnapshotDb.createEmpty` - do we need to repeat it here?

Reply:
Missed that one. Thanks!